IsotropicTextureMat.h

```diff
@@ -29,10 +29,10 @@ class RealFunc;
 class IsotropicTextureMat : public LinIsotropic
 {
 public:
-  //! \brief Default constructor.
+  //! \brief Constructor.
   //! \param[in] ps If \e true, assume plane stress in 2D
   //! \param[in] ax If \e true, assume 3D axi-symmetric material
-  IsotropicTextureMat(bool ps, bool ax);
+  inline IsotropicTextureMat(bool ps, bool ax);
```

Comment:
No, I did not mean it that way, but to write the (empty) function body in the header:
```
IsotropicTextureMat(bool ps, bool ax) : LinIsotropic(ps,ax) {}
```